Jenkinsfile

```diff
@@ -108,9 +108,13 @@ pipeline {
 									cp $WORKSPACE/test/blockchain.db.gz $WORKSPACE/$BRANCH_NAME/dev_blockchain.db.gz
 									cd $WORKSPACE/$BRANCH_NAME
 									cp .env.development .env
+
+									# random port assignment
+									yq --yaml-output '.services.lisk.ports[0]="${ENV_LISK_HTTP_PORT}"|.services.lisk.ports[1]="${ENV_LISK_WS_PORT}"' docker-compose.yml |sponge docker-compose.yml
+
 									LISK_VERSION=1.1.0-alpha.8 make coldstart
 									export CYPRESS_baseUrl=http://127.0.0.1:300$N/#/
-									export CYPRESS_coreUrl=http://127.0.0.1:$( docker-compose port lisk 4000 |cut -d ":" -f 2 )
+									export CYPRESS_coreUrl=http://127.0.0.1:$( docker-compose port lisk $(ENV_LISK_HTTP_PORT) |cut -d ":" -f 2 )
```

Comment:
This will not work: `ENV_LISK_HTTP_PORT` is not defined in this context and the right syntax would be without parentheses.

Reply:
indeed... thanks